packages/phpstan-rules/packages/nette/tests/Rules/NoNetteRenderUnusedVariableRule/Source/used_in_inline_macro.latte

```diff
@@ -0,0 +1,3 @@
+<div n:if="$value">
```

Comment:
hese values are not standalone tokens, but strings that are parsed in Compiler via recursive calls. This is a very bad design.

Thus it's not possible to analyse the variables until the whole latte is compiled to PHP.

Twig handles this via nested node tree, that is easy to traverse and work with :+1: